packages/react-interactions/events/src/dom/Focus.js

```diff
@@ -538,6 +556,23 @@ const focusWithinResponderImpl = {
     props: FocusWithinProps,
     state: FocusState,
   ): void {
+    if (event.type === 'activeelementblur') {
+      const onActiveElementBlur = (props.onActiveElementBlur: any);
+      if (isFunction(onActiveElementBlur)) {
+        const syntheticEvent = createFocusEvent(
+          context,
+          'activeelementblur',
```

Comment:
Why do users of this responder need to know about this custom event rather than receiving the `blur` event that isn't reliably dispatched by browsers when the element is removed?